<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSkeletalRepresentationInitializerModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerSkeletalRepresentationInitializerModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Forward flow</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lb_title">
          <property name="text">
           <string>Input mesh file:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_input_file_path">
          <property name="text">
           <string>input a vtk file(*.vtk)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SelectInputButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Max iteration #:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLSliderWidget" name="sl_max_iter" native="true">
          <property name="decimals" stdset="0">
           <number>0</number>
          </property>
          <property name="pageStep" stdset="0">
           <double>10.000000000000000</double>
          </property>
          <property name="minimum" stdset="0">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum" stdset="0">
           <double>9999.000000000000000</double>
          </property>
          <property name="value" stdset="0">
           <double>500.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Step size in flow(double dt):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLSliderWidget" name="sl_dt" native="true">
          <property name="decimals" stdset="0">
           <number>6</number>
          </property>
          <property name="singleStep" stdset="0">
           <double>0.010000000000000</double>
          </property>
          <property name="value" stdset="0">
           <double>0.001000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Smooth amount:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLSliderWidget" name="sl_smooth_amount" native="true">
          <property name="decimals" stdset="0">
           <number>3</number>
          </property>
          <property name="singleStep" stdset="0">
           <double>0.010000000000000</double>
          </property>
          <property name="maximum" stdset="0">
           <double>2.000000000000000</double>
          </property>
          <property name="value" stdset="0">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Output mesh every # iterations:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="qMRMLSliderWidget" name="sl_freq_output" native="true">
          <property name="decimals" stdset="0">
           <number>0</number>
          </property>
          <property name="minimum" stdset="0">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum" stdset="0">
           <double>9999.000000000000000</double>
          </property>
          <property name="value" stdset="0">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_one_step_flow">
        <property name="text">
         <string>Flow step by step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_flow">
        <property name="text">
         <string>Flow to the end</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_inkling_flow">
        <property name="text">
         <string>Flow with anti-aliasing curvature</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton2" native="true">
     <property name="text" stdset="0">
      <string>Backward flow</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_31">
      <item>
       <widget class="QPushButton" name="btn_back_flow">
        <property name="text">
         <string>Generate S-rep for object</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLSliderWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
